### PR TITLE
SPCTR-2927 Resolving reusable block client id issue in 7 core blocks

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
Fix: Page gets stuck/frozen when more than one reusable block is used on the page.
Fixed in blocks: 
advanced-heading
buttons
buttons-child
call-to-action
countdown
icon
image
info-box

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/image/image.js [hunk 1]
```diff
@@ -89,7 +90,11 @@ export default function Image( {
 		const naturalWidth = imageRef.current?.naturalWidth || loadedNaturalWidth || undefined;
 		// eslint-disable-next-line
```
Comment (aapatre) [inline src/blocks/image/image.js:91]: Can we have a more specific eslint-disable (like `no-console` or whatever the error is?

Comment (rahulvarma722) [reply src/blocks/image/image.js]: Added

File: blocks-config/uagb-controls/addInitialAttr.js [hunk 2]
```diff
@@ -1,8 +1,32 @@
 import { useEffect } from '@wordpress/element';
+
+const getAllBlocks = () => wp.data.select( 'core/block-editor' ).getBlocks();
```
Comment (adi3890) [inline blocks-config/uagb-controls/addInitialAttr.js:3]: should we import select at top as we are following for other components

File: blocks-config/uagb-controls/addInitialAttr.js [hunk 3]
```diff
@@ -62,8 +86,101 @@ const addInitialAttr = ( ChildComponent ) => {
...
+				'uagb/register-first-name',
+				'uagb/register-last-name',
+				'uagb/register-password',
+				'uagb/register-reenter-password',
+				'uagb/register-terms',
+				'uagb/register-username',
+			];
+
+			if( ! REUSABLE_BLOCK_ISSUE_RESOLVED_BLOCKS.includes( name ) ){
+				console.log(' getAllBlocks ', getAllBlocks());
```
Comment (adi3890) [inline blocks-config/uagb-controls/addInitialAttr.js:174]: can you please check for this console.log.

Comment (rahulvarma722) [reply blocks-config/uagb-controls/addInitialAttr.js]: Removed

File: blocks-config/uagb-controls/addInitialAttr.js (hunk 2 above)
Comment (rahulvarma722) [reply blocks-config/uagb-controls/addInitialAttr.js]: Implemented

File: src/blocks/image/image.js [hunk 4]
```diff
@@ -85,11 +86,15 @@ export default function Image( {
 	// width and height. This resolves an issue in Safari where the loaded natural
 	// width and height is otherwise lost when switching between alignments.
 	const { naturalWidth, naturalHeight } = useMemo( () => {
-		// eslint-disable-next-line
+		// eslint-disable-next-line no-shadow
```
Comment (adi3890) [inline src/blocks/image/image.js:89]: can we add these issues. and remove linting issues.

File: src/blocks/countdown/edit.js [hunk 5]
```diff
@@ -76,7 +76,7 @@ const UAGBCountdownEdit = ( props ) => {
 	useEffect( () => {
 		if ( countdownRef ) {
 			setTimeout( () => {
-				UAGBCountdown.editorInit( '.uagb-block-' + clientId.substr( 0, 8 ), attributes, countdownRef.current ); // eslint-disable-line no-undef
+				UAGBCountdown.editorInit( '.uagb-block-' + block_id, attributes, countdownRef.current ); // eslint-disable-line no-undef
```
Comment (adi3890) [inline src/blocks/countdown/edit.js:79]: can you please check for this eslint comment. can this be fixed

Comment (rahulvarma722) [reply src/blocks/countdown/edit.js:79]: eslint comments for UAGBCountdown variable either we can keep this in eslintrc globals variable or we can put this comments. what should we do?

Comment (rahulvarma722) [reply src/blocks/countdown/edit.js]: For now added in eslintrc file globals variable.

**Review by adi3890 — APPROVED** [review]

File: src/blocks/countdown/edit.js (hunk 5 above)
Comment (adi3890) [reply src/blocks/countdown/edit.js]: global

**Pull request merged**